packages/eas-cli/src/credentials/ios/appstore/experimental.ts

```diff
@@ -0,0 +1,26 @@
+import { BundleId, Profile } from '@expo/apple-utils';
+import { boolish } from 'getenv';
+
+export const USE_APPLE_UTILS = boolish('USE_APPLE_UTILS', false);
```

Comment:
```suggestion
export const USE_APPLE_UTILS = Boolean(process.env.USE_APPLE_UTILS);
```
Using `getenv` for this seems to be overkill.

Reply:
`Boolean` is not the same as `getenv.boolish`:
```
> Boolean('false')
true
> Boolean('0')
true
```
With `getenv` both of these are parsed as `false` as one would expect.

Reply:
Well, if the only purpose of the env var is to use it for enabling some experimental feature then the behavior I proposed is perfectly enough.

Reply:
I mean:
- The default behavior is to run eas-cli without any env vars.
- If I wanted to test this feature I would run it like this: `USE_APPLE_UTILS=whatever eas`
- Then, if I wanted to test the original behavior I would run `eas` without any env vars set. I wouldn't even consider setting `USE_APPLE_UTILS=false`. I don't believe anyone would expect this to work.

Reply:
I think it's only temporary while we transition off of Fastlane safely, further `getenv` is a pretty good UX as @fson explained. 
```js
V='1', V='true' === true
V='0', V='false', V='null', V='undefined', V=undefined === false
```

Reply:
🤷‍♂️